### PR TITLE
(python) Fix error related to having an ending double quote in the docstring

### LINKED ISSUE #2247: Python docstring should escape last character if it is a double quote
I'm using quicktype to create python code from my jsonschema. 
One of the descriptions ends in a double quote . 
The generated python code ends up with a syntax error, due to this quote.

Here is a quick reproducer, notice that the `description` of this ends up in a double quote (`"`).

```json
{
  "$schema": "http://json-schema.org/draft-07/schema#",
  "$id": "my-stuff",
  "description": "Cool \"stuff\"",
  "type": "object",
  "properties": {
    "foo": {
      "type": "boolean"
    }
  }
}
```

Which generates this part of a python file
```python
class Stuff:
    """Cool "stuff""""  # <-- This generates an error `Missing closing quote ["]`
    foo: Optional[bool]
```

The last double quote (or all of them) should be escaped to prevent the syntax error or one extra space should be added to prevent this from happening.

### PR DESCRIPTION
**Pull request by josejulio**
Before:
screenshot_2023-04-24T09:42:28

The error is
> Missing closing quote ["]

After:
screenshot_2023-04-24T09:40:51

Fixes #2247

### CODE REVIEW

File: packages/quicktype-core/src/language/Python.ts [hunk 1]
```diff
@@ -275,7 +275,14 @@ export class PythonRenderer extends ConvenienceRenderer {
 
     protected emitDescriptionBlock(lines: Sourcelike[]): void {
         if (lines.length === 1) {
-            this.emitLine('"""', lines[0], '"""');
+            const docstring = modifySource(content => {
+                if (content.length > 0 && content[content.length - 1] === '"') {
```
Comment (dvdsgl) [inline packages/quicktype-core/src/language/Python.ts:279]: Try `endsWith`?

**Comment by dvdsgl** [comment]
Thanks! One small change suggested.

**Comment by josejulio** [comment]
Thanks @dvdsgl ! I will update it once the suggestion is available. I didn't see any, maybe it is still in the review queue?

**Review by dvdsgl — CHANGES_REQUESTED** [review]
Use endsWith

File: packages/quicktype-core/src/language/Python.ts (hunk 1 above)
Comment (josejulio) [reply packages/quicktype-core/src/language/Python.ts:279]:
```suggestion
                if (content.endsWith('"')) {
```

**Pull request merged**